django/contrib/postgres/fields/ranges.py

```diff
@@ -132,6 +133,44 @@ def db_type(self, connection):
 RangeField.register_lookup(lookups.Overlap)
 
 
+class DateTimeRangeContains(models.Lookup):
```

Comment:
We could point at psycopg/psycopg2#231 and psycopg/psycopg2#313 but given their resolution I don't think it's worth it. What about _Lookup for Date/DateTimeRange containment providing explicit rhs type hints to prevent a ProgrammingError raised by PostgreSQL_?